Makefile.custom.mk

```diff
@@ -66,8 +66,31 @@ lint-imports: goimports ## Run go vet against code.
 create-acceptance-cluster: kind
 	CLUSTER=$(CLUSTER) IMG=$(IMG) ./scripts/ensure-kind-cluster.sh
 
+.PHONY: deploy-capg-crds
+deploy-capg-crds: kind
+	KUBECONFIG="$(KUBECONFIG)" CLUSTER=$(CLUSTER) IMG=$(IMG) ./scripts/install-crds.sh
+
+.PHONY: create-test-secrets
+create-test-secrets: kind
+	CLUSTER=$(CLUSTER) IMG=$(IMG) ./scripts/create-test-secrets.sh
+
 .PHONY: deploy-acceptance-cluster
-deploy-acceptance-cluster: docker-build create-acceptance-cluster deploy
+deploy-acceptance-cluster: docker-build create-acceptance-cluster deploy-capg-crds create-test-secrets deploy-crds-on-workload deploy-on-workload-cluster deploy
+
+.PHONY: deploy-crds-on-workload
+deploy-crds-on-workload: kind
+	KUBECONFIG="$(HOME)/.kube/workload-cluster.yaml" CLUSTER=$(CLUSTER) IMG=$(IMG) ./scripts/install-crds.sh
+
+.PHONY: deploy-on-workload-cluster
+deploy-on-workload-cluster: manifests render
+	 helm upgrade --install \
+	  --kubeconfig="$(HOME)/.kube/workload-cluster.yaml" \
+		--namespace giantswarm \
+		--set image.tag=$(IMAGE_TAG) \
+		--set operationMode=onprem \
```

Comment:
`operationMode` doesn't really say what's the purpose of the parameter. Maybe something like useLocalCredentials=true` would be better?

Anyway, why do we need this in the first place? When aren't we using the credentials from a secret?

Reply:
It does use a secret. When outside of a GCP environment, the client looks for the `GOOGLE_APPLICATION_CREDENTIALS` whose value should be a path to a `json` file that contains the credentials needed. Helm looks at this parameter to know whether or not to add this to the deployment.

Reply:
> When outside of a GCP environment

Does this mean that we're using the default VM service account when running on GCP?

Reply:
No, if I understood it correctly it would use the attached service account, not necessarily default

Reply:
What I mean was that it would use the SA on the VM, in which case I'd say we shouldn't rely on it and should always require the credentials be set.